pkgs/applications/science/electronics/linux-gpib/linux-gpib-kernel/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchurl
+, kernel
+, pahole
+}:
+
+stdenv.mkDerivation rec {
+  pname = "linux-gpib";
+  version = "4.3.6";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/linux-gpib/files/linux-gpib%20for%203.x.x%20and%202.6.x%20kernels/${version}/${pname}-${version}.tar.gz";
+    hash = "sha256-Gze4xrvkhEgn+J5Jhrycezjp2uhlD1v6aX0WGv4J2Jg=";
+  };
+
+  unpackPhase = ''
+    tar xf $src
+    tar xf ${pname}-${version}/${pname}-kernel-${version}.tar.gz
+  '';
+
+  postPatch = ''
+    sed -i 's@/sbin/depmod -A@@g' Makefile
+  '';
+
+  sourceRoot = "${pname}-kernel-${version}";
```

Comment:
```suggestion
  sourceRoot = "linux-gpib-kernel-${version}";
```

Reply:
Please check if updated code also achieves the same.